ssh/tailssh/tailssh.go

```diff
@@ -337,6 +337,24 @@ func (c *conn) fakePasswordHandler(ctx ssh.Context, password string) bool {
 	return c.anyPasswordIsOkay
 }
 
+// fakePublicKeyHandler is our implementation of the PublicKeyHandler hook that
+// checks whether the user's public key is correct. But we don't actually use
+// public keys. This exists only for when a client provides an SSH public key
+// because the client is buggy and gets confused by auth type "none" succeeding.
+// We then accept any public key since we've already authenticated & authorized them.
+func (c *conn) fakePublicKeyHandler(ctx ssh.Context, pubKey ssh.PublicKey) error {
+	if err := c.doPolicyAuth(ctx); err != nil {
+		// TODO(maisem/bradfitz): surface the error here.
```

Comment:
I'm not sure what this comment means. It's logged and returned.

Does this mean sending a banner?

Reply:
Yeah, I'm not sure, I carried it over from the old code. I'll remove it.